packages/tools/gulp/tasks/buildScriptIndexes.js

```diff
@@ -12,7 +12,7 @@ async function processDirectory(dir, indexFilePath = null, typeScriptType = null
 
 	const tsFiles = glob.sync('{**/*.ts,**/*.tsx}', {
 		cwd: dir,
-	}).filter(f => `${dir}/${f}` !== indexFilePath);
+	}).filter(f => `${dir}/${f}` !== indexFilePath && !f.endsWith('.test.ts'));
```

Comment:
```suggestion
	}).filter(f => `${dir}/${f}` !== indexFilePath)
	// Exclude Jest test files to prevent them from being imported
	// by index.ts.
	.filter(f => !f.endsWith('.test.ts'));
```

**(Optional)** A brief comment might be helpful here.

Reply:
I will do that. Thanks for sharing your implementation!